dimensiondata/src/test/java/org/jclouds/dimensiondata/cloudcontrol/internal/BaseDimensionDataCloudControlApiLiveTest.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Properties;
 import java.util.Set;
 
-import static com.google.common.util.concurrent.MoreExecutors.sameThreadExecutor;
+import static com.google.common.util.concurrent.MoreExecutors.newDirectExecutorService;
```

Comment:
@demobox I think this is required because of guava 21 inherited from jclouds/jclouds